resources/views/user/profile/calendars.blade.php

```diff
@@ -43,14 +43,17 @@
 
     <div class="input-group input-group-select2 mb-3">
         <select class="form-control select2-with-input-group" id="inputGroupSelect02">
+        @php($first_option=null)
```

Comment:
What does this do?

Reply:
There's a template error that you see if you go to the Profile page on a system which is empty, as my test system sometimes is.  If there are no areas then first_option is not set and causes an error.  I slapped this in to make it load.